src/interp/interp.h

```diff
@@ -74,7 +74,9 @@ namespace interp {
   /* we tried to get an export by name that doesn't exist */                \
   V(UnknownExport, "unknown export")                                        \
   /* the expected export kind doesn't match. */                             \
-  V(ExportKindMismatch, "export kind mismatch")
+  V(ExportKindMismatch, "export kind mismatch")                             \
+  /* instruction not implemented in the interpreter */                      \
+  V(NotImplemented, "not implemented")
```

Comment:
Interesting.. is this really the first time that we've added instructions and left them as unimplemented in the interpreter?

Reply:
I think in the past I'd left them as WABT_UNREACHABLE, but that's dangerous, since it ends up being compiled to "this never happens", not an abort. Throwing an error is nicer, I think.